Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -499,30 +492,27 @@ extension ChatChannel {
                     context: context
                 )?
                 .relationshipAsModel(depth: depth)
-        }
+        }()
 
-        let fetchWatchers: () -> [ChatUser] = {
-            UserDTO
-                .loadLastActiveWatchers(cid: cid, context: context)
-                .compactMap { try? $0.asModel() }
-        }
-
-        let fetchMembers: () -> [ChatChannelMember] = {
-            MemberDTO
-                .loadLastActiveMembers(cid: cid, context: context)
-                .compactMap { try? $0.asModel() }
-        }
+        let watchers = UserDTO.loadLastActiveWatchers(cid: cid, context: context)
+            .compactMap { try? $0.asModel() }
+        
+        let members = MemberDTO.loadLastActiveMembers(cid: cid, context: context)
+            .compactMap { try? $0.asModel() }
```

Comment:
We should do what we do on `ThreadList` with the `latestReplies`, to avoid DB queries. Just a note, since this PR is out of scope